integration/tests/cook/test_basic.py

```diff
@@ -29,6 +29,12 @@ def test_basic_submit(self):
         job = util.wait_for_job(self.cook_url, job_uuid, 'completed')
         self.assertEqual('success', job['instances'][0]['status'])
 
+    def test_disable_mea_culpa(self):
```

Comment:
Do we also need tests for:

- explicitly setting the field to false
- not setting the field and asserting the default
- checking the actual effect of the field

?